homeassistant/components/drop_connect/switch.py

```diff
@@ -0,0 +1,127 @@
+"""Support for DROP switches."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+import logging
+from typing import Any
+
+from homeassistant.components.switch import SwitchEntity, SwitchEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import (
+    CONF_DEVICE_TYPE,
+    DEV_FILTER,
+    DEV_HUB,
+    DEV_PROTECTION_VALVE,
+    DEV_SOFTENER,
+    DOMAIN,
+)
+from .coordinator import DROPDeviceDataUpdateCoordinator
+from .entity import DROPEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+ICON_VALVE_OPEN = "mdi:valve-open"
+ICON_VALVE_CLOSED = "mdi:valve-closed"
```

Comment:
```suggestion
ICON_VALVE_OPEN = "mdi:valve-open"
ICON_VALVE_CLOSED = "mdi:valve-closed"
ICON_VALVE_UNKNOWN= "mdi:valve"

ICON_VALVE = {
    0: ICON_VALVE_CLOSED,
    1: ICON_VALVE_OPEN,
} 
```

Reply:
Updated in 00255a784e537b659831511f69d179a6c646d4bc, though the `ICON_VALVE` dict needed to be keyed on bools and include the None type.